lighthouse-core/gather/gatherers/full-page-screenshot.js

```diff
@@ -119,8 +103,6 @@ class FullPageScreenshot extends Gatherer {
           return {
             width: document.documentElement.clientWidth,
             height: document.documentElement.clientHeight,
-            screenWidth: window.screen.width,
```

Comment:
Was this necessary before this patch? Looks like `screenWidth` and `screenHeight` are not change in this gatherer.

Reply:
fair. i meant to include it in #11688 instead. just moved it there.